#accepts "robot", "picker", "virtual_picker"
string type
#The name of the robot, used to define the object localisation topic
string name
#accepts "none", "red", "green", "blue", "white", "black"
string optional_color